File: app/src/main/java/ru/magflayer/spectrum/presentation/common/android/BaseViewHolder.java
package ru.magflayer.spectrum.presentation.common.android;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import butterknife.ButterKnife;

public abstract class BaseViewHolder extends RecyclerView.ViewHolder {
    public BaseViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/ru/magflayer/spectrum/presentation/common/android/BaseViewHolder.kt
package ru.magflayer.spectrum.presentation.common.android

import android.view.View
import androidx.recyclerview.widget.RecyclerView

import butterknife.ButterKnife

abstract class BaseViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    init {
        ButterKnife.bind(this, itemView)
    }
}
